src/api/app/models/group.rb

```diff
@@ -187,6 +187,10 @@ def any_confirmed_users?
     !users.where(state: 'confirmed').empty?
   end
 
+  def away?
+    users.recently_seen.empty?
+  end
+
```

Comment:
Please explain the reason of this commit in the body of the commit message. Or merge this commit into the first one. It seems we are creating methods in different classes, but with the same purpose.

Reply:
I think @vpereira is following the [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter)

Reply:
The PR is pretty descriptive, the method itself is clear.. could you please mark it as nitpick?

Reply:
Yes, of course. Moved to nitpick.